example-tests/src/test/java/com/epam/restassured/util/Driver.java

```diff
@@ -0,0 +1,40 @@
+package com.epam.restassured.util;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Created by Peter_Olah1 on 12/10/2015.
+ */
+public class Driver {
+    private static WebDriver driver;
```

Comment:
Generally its not good to have a shared static variable. This would kill parallel test execution because of the shared WD instance.